### ISSUE TITLE
Error when trying to start script "ValueError: invalid literal for int() with base 10: 'None'"

### ISSUE THREAD

**Issue by chrizzler9** [TobiasPankner/Teams-Auto-Joiner#45]
Hi, I am trying to run the script using your last build, running via CMD admin/Windows 10/Edge mode. Running the latest PIP. Any idea why I am getting this error message?

C:\***\Teams-Auto-Joiner-master>python auto_joiner.py
Traceback (most recent call last):
  File "auto_joiner.py", line 636, in <module>
    main()
  File "auto_joiner.py", line 504, in main
    init_browser()
  File "auto_joiner.py", line 173, in init_browser
    browser = Edge(EdgeChromiumDriverManager().install(), options=chrome_options)
  File "C:\Users\***\AppData\Local\Programs\Python\Python38\lib\site-packages\webdriver_manager\microsoft.py", line 45, in install
    return self.download_driver(self.driver)
  File "C:\Users\***\AppData\Local\Programs\Python\Python38\lib\site-packages\webdriver_manager\manager.py", line 44, in download_driver
    path = self.__download_and_save_driver_to_cache(driver, driver_version)
  File "C:\Users\***\AppData\Local\Programs\Python\Python38\lib\site-packages\webdriver_manager\manager.py", line 32, in __download_and_save_driver_to_cache
    response = download_driver(driver.get_url(driver_version))
  File "C:\Users\***\AppData\Local\Programs\Python\Python38\lib\site-packages\webdriver_manager\utils.py", line 58, in download_driver
    log(f"Trying to download new driver from {url}")
  File "C:\Users\***\AppData\Local\Programs\Python\Python38\lib\site-packages\webdriver_manager\logger.py", line 12, in log
    level = int(log_level)
ValueError: invalid literal for int() with base 10: 'None'

**Comment by TobiasPankner** [comment]
This seems to be an issue with the webdriver_manager package.  
I've looked into it and I think that I have found a workaround.  
Try setting the enviroment variable   WDM_LOG_LEVEL=1  
this should solve the problem.

**Comment by TobiasPankner** [comment]
Please try running the bot like this:  
set WDM_LOG_LEVEL=1 && python auto_joiner.py  
in cmd

**Comment by chrizzler9** [comment]
That solved the issue, thanks. Now edge is starting, however I am getting this error in the edge console: C:\program files...\edge\application\msedge.exe. Maybe its missing permissions? Just let me know if any error is probably related to my configuration versus a potential issue with the script.

[3576:9988:1102/111729.380:ERROR:file_io_win.cc(180)] CreateFile C:\Users\****\AppData\Local\Temp\scoped_dir2852_1758677056\Crashpad\settings.dat: The system cannot find the path specified. (0x3)
[3576:9988:1102/111729.380:ERROR:file_io_win.cc(180)] CreateFile C:\Users\****\AppData\Local\Temp\scoped_dir2852_1758677056\Crashpad\settings.dat: The system cannot find the path specified. (0x3)
[3576:9988:1102/111729.496:ERROR:file_io_win.cc(180)] CreateFile C:\Users\****\AppData\Local\Temp\scoped_dir2852_1758677056\Crashpad\settings.dat: The system cannot find the path specified. (0x3)
[9552:14892:1102/111729.598:ERROR:file_io_win.cc(180)] CreateFile settings.dat: Access is denied. (0x5)

DevTools listening on ws://127.0.0.1:56446/devtools/browser/0d321e3d-3a9d-4bfc-b278-bab47b5fe5a2
[9864:12844:1102/111729.850:ERROR:file_io_win.cc(180)] CreateFile settings.dat: Access is denied. (0x5)
[580:13676:1102/111733.450:ERROR:file_io_win.cc(180)] CreateFile settings.dat: Access is denied. (0x5)
[16740:3500:1102/111734.753:ERROR:file_io_win.cc(180)] CreateFile settings.dat: Access is denied. (0x5)

**Comment by TobiasPankner** [comment]
This seems to be an issue witg edge itself. Try starting cmd with admin privileges. I also strongly recommend to use Google Chrome instead.

**Comment by chrizzler9** [comment]
All right. Ran it with admin permission before. Just if you encounter anything like it in the future, I am running Edge Version 86.0.622.58 (Official build) (64-bit) with a few standard plugins like ABP. All I am getting from edge after starting is " yourprofile@mail.com has signed in on this device, so we've also signed you in to Microsoft Edge. with the crash messages I posted above. The script worked 6 weeks ago with Edge so I guess something must have changed. Anyway, great work, thank you for that & kind Regards
